clients/python/cac_client/cac_client/client.py

```diff
@@ -66,16 +66,24 @@ class CacClient:
     rust_lib.cac_get_default_config.argtypes = [ctypes.c_char_p, ctypes.c_char_p]
     rust_lib.cac_get_default_config.restype = ctypes.c_char_p
 
-    def __init__(self, tenant_name: str, polling_frequency: int, cac_host_name: str):
+    def __init__(self, tenant_name: str, polling_frequency: int, cac_host_name: str, cache_max_capacity: int| None = None, cache_ttl: int| None = None, cache_tti: int| None = None):
         if not tenant_name or not cac_host_name:
             raise ValueError("tenantName cannot be null or empty")
 
         self.tenant = tenant_name
         self.polling_frequency = polling_frequency
         self.cac_host_name = cac_host_name
+        self.cache_max_capacity = cache_max_capacity
+        self.cache_ttl = cache_ttl
+        self.cache_tti = cache_tti
+
+        max_capacity_ptr = ctypes.pointer(ctypes.c_int(cache_max_capacity)) if cache_max_capacity is not None else ctypes.POINTER(ctypes.c_int)()
```

Comment:
can this simply not be written as:
```suggestion
        max_capacity_ptr = ctypes.pointer(ctypes.c_int(cache_max_capacity))
```
sending `None` as param should be same as not sending any param, right ?

Reply:
@ayushjain17 we cannot write in that way as cache_max_capacity can be None , we cannot do ctypes.pointer(ctypes.c_int(cache_max_capacity))

Reply:
no, my question was, 
is `ctypes.pointer(ctypes.c_int(None))` and `ctypes.pointer(ctypes.c_int())` different ?
aren't these two the same ?

Reply:
no these two are not same, c_int() expects an integer value
also in None case it is `ctypes.POINTER(ctypes.c_int)()` not `ctypes.pointer(ctypes.c_int())`

Reply:
oh right, missed that subtle difference